### ISSUE TITLE
Color contrast does not meet minimum requirements - Clear selected items button

### ISSUE THREAD

**Issue by elynema** [avalonmediasystem/avalon#5628]
# Description 

The 'Clear selected items' button the Selected Items page does not meet minimum color contrast requirements. 

<img width="1296" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBeThzQXc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--0fc2a3583a273d27eb9e72730a46951ed35258e5/image.png" alt="image.png" />

Recommendation from SiteImprove: Increase the color contrast to 4.5:1 or higher. 

Current color contrast is 3.68:1. 
<img width="347" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBekVzQXc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--2e18b115fa340d9d8c92caad2238500da2332ff4/image.png" alt="image.png" />

Note: This was found by the SiteImprove browser plugin, not the SiteImprove crawler, so this does not *have* to be pushed out by 2/8.

# Done Looks Like
- [ ] Pick a color replacement for the current red warning button style
- [ ] Change CSS for "super bright red button" styling that applies to buttons in the application for deletion

**Comment by Dananji** [comment]
Fix in the PR:

Value for `$danger` SCSS variable is changed from `#F44336` to `#D14242`.

The value `#D14242` was derived by using https://webaim.org/resources/contrastchecker/, with values for foreground color set to `#FFFFFF` and background color set to `#F44336` (old color) and adjusting the luminance till contrast ratio reached 4.59:1 (in compliance with WCAG 2.0 level AA).

Before:
<img width="1267" alt="Screenshot 2024-04-12 at 4 26 07 PM" src="https://github.com/avalonmediasystem/avalon/assets/1331659/725c66f2-9a95-4f7c-9996-360edd5be955">

After:
<img width="1267" alt="Screenshot 2024-04-12 at 4 29 27 PM" src="https://github.com/avalonmediasystem/avalon/assets/1331659/f78fce02-0287-4fc3-a838-987e581f051e">

**Comment by joncameron** [comment]
Much better! Easier on the eyes and to read.

**Comment by elynema** [comment]
@joncameron One thing we mentioned in person on Monday when you were out was wondering whether this color was too similar to the crimson we use for buttons and highlighting in our normal color scheme?

**Comment by joncameron** [comment]
It is similar; we could change this to match the Rivet danger color instead: https://rivet.iu.edu/components/button/ #df3603, either in core or local prod instances or both since it seems to also meet contrast requirements.

**Comment by joncameron** [comment]
Comparison:

Before: `#F44336`
<img width="1267" alt="Screenshot 2024-04-12 at 4 26 07 PM" src="https://github.com/avalonmediasystem/avalon/assets/1331659/725c66f2-9a95-4f7c-9996-360edd5be955">

After: `#D14242`
<img width="1267" alt="Screenshot 2024-04-12 at 4 29 27 PM" src="https://github.com/avalonmediasystem/avalon/assets/1331659/f78fce02-0287-4fc3-a838-987e581f051e">

Final: `#df3603`
